gen/config/downloads.yaml

```diff
@@ -0,0 +1,34 @@
+---
+-
+  url: "http://www.unicode.org/Public/{}/ucd/DerivedAge.txt"
```

Comment:
Since Rust doesn't have run-time format strings, it's currently handled by a simple `replace("{}", version)`. It would make sense to have it be named though, and that will just change the search string.